DOCUMENTATION.md

```diff
@@ -231,6 +231,40 @@ If the request is not successful, the client returns a `NotificationClientExcept
 |`429`|`[{`<br>`"error": "TooManyRequestsError",`<br>`"message": "Exceeded send limits (LIMIT NUMBER) for today"`<br>`}]`|Refer to [service limits](#service-limits) for the limit number|
 |`500`|`[{`<br>`"error": "Exception",`<br>`"message": "Internal server error"`<br>`}]`|Notify was unable to process the request, resend your notification|
 
+##Send a document by email
```

Comment:
Add a space between ## and Send - sometimes we get an error in the tech doc tool when there aren't spaces between the # and the title